.github/workflows/docker-push.yml

```diff
@@ -0,0 +1,92 @@
+name: Docker Push
+
+on:
+  push:
+    branches: [ "main" ]
+    tags: [ "v*.*.*" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
I think you have to delete this after doing experiment on the github actions
Just trigger when push to branch main
and do not push to docker hub when no version changed

Reply:
agree